packages/react-storage/src/components/StorageBrowser/context/actions/listLocationItemsAction.ts

```diff
@@ -68,8 +103,9 @@ export async function listLocationItemsAction(
 
   const result = [
     ...(refresh ? [] : prevState.result),
-    ...parseResultItems(output.items),
-  ];
+    ...parseResultItems(output.items, path),
+    ...parseResultExcludedPaths(output.excludedSubpaths, path),
```

Comment:
Probably okay to do later but might make sense to abstract a util that calls both of these so it can be unit tested in isolation?

Reply:
done